changelog/18610.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+auth: named login MFA methods
```

Comment:
This changelog isn't going to make a log of sense to most of our users.

Reply:
I changed it to the following. Please let me know what you think? 
```
auth: Adding support for an additional name parameter for login MFA methods.
Then, this name can be used in both single-phase and two-phase login requests instead of an MFA method ID.

cli/api: providing consistency in using "passcode=" in login requests that are subject to login-MFA.
```

Reply:
That's fine, though I might go a bit more succinct, e.g.: Allow naming login MFA methods and using those names instead of IDs in satisfying MFA requirement for requests.  Make passcode arguments consistent across login MFA method types.